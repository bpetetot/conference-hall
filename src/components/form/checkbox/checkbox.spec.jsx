/* eslint-env jest */
import { shallow } from 'enzyme'
import React from 'react'
import Checkbox from './index.js'

describe('components/checkbox', () => {
  it('should render with children', () => {
    expect(shallow(<Checkbox name="name" label="label" />)).toMatchSnapshot()
  })
})
