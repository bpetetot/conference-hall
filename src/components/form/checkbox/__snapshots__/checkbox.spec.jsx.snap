// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/checkbox should render with children 1`] = `
<div
  className="form-checkbox"
>
  <input
    defaultChecked={false}
    id="name-false"
    label="label"
    name="name"
    type="checkbox"
    value={false}
  />
  <label
    htmlFor="name-false"
  >
    label
  </label>
</div>
`;
